src/components/List/List.js

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
 import { FlatList } from 'react-native';
-import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import ListItem from '../ListItem';
-
-const List = ({ todos }) => {
-  return (
-    <FlatList
-      keyExtractor={todo => todo.id}
-      data={todos}
-      renderItem={({ item }) => <ListItem todo={item} />}
-    />
-  );
+const List = ({ elems, render }) => {
+  return <FlatList keyExtractor={elem => elem.id} data={elems} renderItem={render} />;
```

Comment:
This Component is too simple, you should use the FlatList directly.

Reply:
Done on the next commit.